pm_yaml_config/pm_yaml_config.xml

```diff
@@ -0,0 +1,126 @@
+        echo $calculator_cond.calculator &&
+        #if str($calculator_cond.calculator)=="uep":
+            printf "uep_gw_factor: $calculator_cond.uep_gw_factor \nuep_chden: $calculator_cond.uep_chden \n">>outputx.yaml &&
+        #else if str($calculator_cond.calculator)=="castep": 
+            printf "castep_command: $calculator_cond.castep_command \ncastep_param: $calculator_cond.castep_param \nmu_symbol: $calculator_cond.mu_symbol \n">>outputx.yaml &&
+        #else if str($calculator_cond.calculator)=="dftb+": 
+            printf "dftb_set: $calculator_cond.dftb_set \ndftb_pbc: $calculator_cond.dftb_pbc \n">>outputx.yaml &&
+        #else if str($calculator_cond.calculator)=="all": 
+            printf "uep_gw_factor: $calculator_cond.uep_gw_factor \nuep_chden: $calculator_cond.uep_chden \ncastep_command: $calculator_cond.castep_command \ncastep_param: $calculator_cond.castep_param \nmu_symbol: $calculator_cond.mu_symbol \ndftb_set: $calculator_cond.dftb_set \ndftb_optionals: $calculator_cond.dftb_optionals \ndftb_pbc: $calculator_cond.dftb_pbc \n">>outputx.yaml &&
+        #end if 
+        ln -s outputx.yaml output.yaml
+    ]]></command>
+    <inputs>
+        <param type="float" name="poisson_r" value="0.8" label="poisson radius" help=" Poisson sphere radius to use for random generation. No two starting muon positions will be closer than this distance. Smaller values make for bigger structure sets."/>
+        <param type="text" name="struct_name" label="name" help="name to call the folder for containing each structure. This name will be postfixed with a unique number, e.g. struct_001."/>
+        <param type="select" name="charged" label="charged muon" value="true" help="if True, the muon will be considered charged instead of a muonium with an accompanying electron. Must be True for UEP calculations.">
+            <option value="true">true</option>
+            <option value="false">false</option>
+        </param>
+        <param type="integer" name="geom_steps" value="300" label="geometry optimization steps" help="maximum number of geometry optimisation steps."/>
+        <param type="float" name="vdw_scale" value="0.25" label="Van der Waal scale" help="Van der Waals scaling factor to use when generating muon sites to avoid existing atoms. Smaller values will allow muons to get closer to the other ions."/>
+        <param type="text" name="out_folder" value="muon-airss-out" label="output folder name" help="name to call the output folder used to store the input files that the script generates."/>
+        <param type="float" name="geom_force_tol" value="0.05" label="geometry optimization tolerance" help="tolerance on geometry optimisation in units of eV/Armstrong."/>
```

Comment:
It's Angstrom, not Armstrong! But I think muon people will recognise the symbol fine (AA and Å represent the same thing, AA is the ASCII-friendly version but we aren't limited to that here)
```suggestion
        <param type="float" name="geom_force_tol" value="0.05" label="geometry optimization tolerance" help="tolerance on geometry optimisation in units of eV/Å."/>
```